### ISSUE TITLE
FEDformer model training results are not ideal

### ISSUE THREAD

**Issue by fizzking** [MAZiqing/FEDformer#43]
I have downloaded the ETTh1 data set and one of the weather data sets, and run the same data set with FEDformer, Autoformer, Informer and Transformer provided by you respectively, but the effect is different from that shown in your paper. The result of my running is as follows: I made the input length and the prediction length both 96. Is where I did not adjust the hyperparameter?
![image](https://user-images.githubusercontent.com/111410702/223401006-d827db59-e039-4245-854a-fd2db3aec675.png)

**Comment by tianzhou2011** [comment]
python -u run.py   --is_training 1   --root_path ./dataset/ETT-small/   --data_path ETTh1.csv   --task_id ETTh1   --model FEDformer   --data ETTh1   --features M   --seq_len 96   --label_len 48   --pred_len 96   --e_layers 2   --d_layers 1   --factor 3   --enc_in 7   --dec_in 7   --c_out 7   --des 'Exp'   --d_model 512   --itr 3
Just Run the above comand minutes ago, The results for running ETTh1 using Fedformer were the following. Not sure what's wrong in your case. 
![image](https://user-images.githubusercontent.com/7770939/223599796-40410e0a-d4af-42b1-901d-ee7b65b0aca2.png)

**Comment by fizzking** [comment]
Thank you for your reply. I used your code to run and got the same ideal result, but I found that the groundtruth and prediction result drawn was strange and there would be a negative number, while the target of my data set was weather visibility ' vs', which would not have a negative number, but there was  negative numbers in the prediction result and the value was relatively small. What's the problem？Below is the data set I used and the truth, prediction, drawn and printed.
![image](https://user-images.githubusercontent.com/111410702/224706041-96134b14-b322-4919-b40a-8a06420af11f.png)
![image](https://user-images.githubusercontent.com/111410702/224706009-8976434d-f2f5-4b18-8cb3-66d5ba1e23df.png)
![image](https://user-images.githubusercontent.com/111410702/224706028-e13eb096-6051-4b90-98e0-5b796cc63f0d.png)
